src/app/features/tech-record/tech-record.component.html

```diff
@@ -0,0 +1,8 @@
+<ng-container *ngIf="vehicleTechRecord$ | async as vehicleTechRecord; else elseBlock">
+  <app-vehicle-technical-record [vehicleTechRecord]="vehicleTechRecord || undefined"></app-vehicle-technical-record>
+</ng-container>
+<ng-template #elseBlock>
+  <ng-container *ngIf="(spinnerService.showSpinner$ | async) === false">
```

Comment:
Is it possible to label this `ng-container` as the `#elseBlock` to save having the `ng-template` declaration?